packages/web-app-files/src/components/ActionMenuItem.vue

```diff
@@ -14,6 +14,13 @@
         alt=""
         class="oc-icon oc-icon-m"
       />
+      <oc-img
+        v-else-if="action.icon && action.icon.startsWith('https://')"
```

Comment:
What about a regex match for `/^https?:\/\//i` so catch both `http://` and `https://`?

Reply:
Could be, not sure if it's worth the struggle when https is kinda omnipresent by now?

Reply:
IMO it is worth a helper 🤷 I'll take care.